framework/doc/content/source/kernels/ADMatCoupledTimeDerivative.md

```diff
@@ -0,0 +1,25 @@
+# ADMatCoupledTimeDerivative
```

Comment:
"MatCoupled" isnt as clear as "CoefCoupled" or "ScaledCoupled" unforuntely. 

Wthiout looking I would have assumed it contributes dMatProp/dt to the equation for 'variable'